modules/virt-creating-a-service-from-a-running-virtual-machine.adoc

```diff
@@ -0,0 +1,191 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc
+
+[id="virt-creating-a-service-from-a-running-virtual-machine_{context}"]
+
+= Creating a service from a running virtual machine
+
+You can create and expose a service from a running virtual machine.
+
+Once the virtual machine is running, create a service object to expose the service.
+Connect to the service by specifying the service name.
+
+The service exposes the host name of the virtual machine to the internal cluster,
+using the `ClusterIP` service, or externally, using the `NodePort` or `LoadBalancer`
+service.
+
+Because the service is scoped by the namespace to which it is defined, ensure
+that you reference the correct namespace in all YAML definitions.
+
+This procedure presents an example of how to create and expose a service object
+of `type: ClusterIP` as a virtual machine-backed service.
+
+[NOTE]
+====
+ClusterIP is the default service `type`, if the service `type` is not specified.
+====
+
+.Procedure
+
+. Start the virtual machine if it is not already running.
```

Comment:
I would skip this step. There's no point ij starting the VM before adding the service-label ("special: key").

Reply:
Skipped. Thanks.